src/main/osd/osd_elements.c

```diff
@@ -1962,7 +2017,7 @@ static void osdDrawSingleElement(displayPort_t *osdDisplayPort, uint8_t item)
     element.attr = DISPLAYPORT_SEVERITY_NORMAL;
 
     // Call the element drawing function
-    if ((item >= OSD_SYS_GOGGLE_VOLTAGE) && (item < OSD_ITEM_COUNT)) {
+    if ((item >= OSD_SYS_GOGGLE_VOLTAGE) && (item < OSD_GPS_LAP_TIME_CURRENT)) {
```

Comment:
Sorry to be picky, having suggested this change, but it would be cleaner expressed as `<= OSD_SYS_FAN_SPEED`.